actors/evm/src/interpreter/instructions/state.rs

```diff
@@ -0,0 +1,22 @@
+use crate::U256;
+use {
+    crate::interpreter::{ExecutionState, StatusCode, System},
+    fil_actors_runtime::runtime::Runtime,
+    fvm_ipld_blockstore::Blockstore,
+};
+
+#[inline]
+pub fn balance<'r, BS: Blockstore, RT: Runtime<BS>>(
+    _state: &mut ExecutionState,
+    _platform: &'r System<'r, BS, RT>,
+) -> Result<(), StatusCode> {
+    todo!("requires syscall")
+}
+
+#[inline]
+pub fn selfbalance<'r, BS: Blockstore, RT: Runtime<BS>>(
+    state: &mut ExecutionState,
+    platform: &'r System<'r, BS, RT>,
+) {
+    state.stack.push(U256::from(&platform.rt.current_balance()))
```

Comment:
In case anyone else is wondering, I've confirmed that Ethereum's token precision is "atto", same as Filecoin.

Reply:
I'll add a comment here.